python/py-canmatrix/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        ebroecker canmatrix 0.6
+
+name                py-canmatrix
+categories-append   devel
+platforms           darwin
+license             BSD-2-Clause
```

Comment:
I don't think this is a valid license from our list. See: https://trac.macports.org/browser/trunk/base/portmgr/jobs/port_binary_distributable.tcl

BSD would be valid.

Reply:
Done

Reply:
Generally: it is up to the person reviewing to resolve a conversation, at the point where they are happy, not up to the person who is submitting code for review. In this case I'm happy but you shouldn't have assumed.

Reply:
Ok, it's clear now